operations/validate_csm_health.md

```diff
@@ -594,6 +594,10 @@ This section can be run on any NCN or the PIT node.
    > waiting state trying to set up volume mounts. See the
    > [UAI Troubleshooting](#64-uasuai-troubleshooting) section for more information.
 
+   > **IMPORTANT:** If the site is configured to use the CHN, and the high speed network has not been
```

Comment:
The style checker doesn't like this block quote being separate from the above block quote. I think you could merge them to fix the checker, or if you really want it to be separate maybe add another new line (assuming that doesn't make the style-checker fail again.

Using a list in the block quote would fix it (e.g. merging the above and new block quote into a single block quote that listed each item).